worlds/shivers/Options.py

```diff
@@ -47,13 +56,41 @@ class EarlyLightning(Toggle):
     """
     display_name = "Early Lightning"
 
+class LocationPotPieces(Choice):
+    """Chooses where pot pieces will be located within the multiworld.
+    - Own World: Pot pieces will be located within your own world
+    - Different World: Pot pieces will be located in another world
+    - Any World: Pot pieces will be located in any world"""
```

Comment:
Some of these should have their closing `"""` on a separate line
```suggestion
    - Any World: Pot pieces will be located in any world
    """
```

Reply:
Basically like how `IxupiCapturesNeeded` is written. (the opening """ should be on their own line as well)

Reply:
All options fixed to have """ on their own line for consistency.